CoreDataAccessView/test/unit/src/au/gov/asd/tac/constellation/views/dataaccess/plugins/clean/SplitNodesPluginNGTest.java

```diff
@@ -96,7 +97,7 @@ public void testQueryWithTypes() throws Exception {
         expResult.set(GraphRecordStoreUtilities.SOURCE + VisualConcept.VertexAttribute.IDENTIFIER, "+123456789");
         expResult.set(GraphRecordStoreUtilities.SOURCE + AnalyticConcept.VertexAttribute.TYPE, AnalyticConcept.VertexType.TELEPHONE_IDENTIFIER);
         expResult.set(GraphRecordStoreUtilities.DESTINATION + VisualConcept.VertexAttribute.IDENTIFIER, "192.168.1.1");
-        expResult.set(GraphRecordStoreUtilities.DESTINATION + AnalyticConcept.VertexAttribute.TYPE, AnalyticConcept.VertexType.URL);
+        expResult.set(GraphRecordStoreUtilities.DESTINATION + AnalyticConcept.VertexAttribute.TYPE, AnalyticConcept.VertexType.HOST_NAME);
```

Comment:
Why set the type to HostName when the identifier is an IP address?

Reply:
I didn't think about this change too much though I didn't think it was a problem until your comment. The reference to URL was wrong and the unit test was failing so I changed it to HOSTNAME which is what that IP was resolving to.  Though it should be IP address so I think there is a bug with Types not resolving correctly @cygnus-x-1 ?

Reply:
I don'r see how the type could be causing issues here - the type should be irrelevant to the operation of this plugin. However, if the type is causing issues - then perhaps there is a schema-related bug that needs resolving?

Reply:
> the type should be irrelevant to the operation of this plugin

Not really @cygnus-x-1, the plugin has to pick a type for the text it splits up.

>then perhaps there is a schema-related bug that needs resolving

Not sure. For the IP 192.168.1.1 there are 3 possible types and using the `AnalyticDominanceCalculator` the one with highest priority is `HOST_NAME` so that's why it's getting picked. Below is the output of the possible types when I run the unit test:

```
Jun 17, 2020 4:34:22 PM au.gov.asd.tac.constellation.graph.schema.type.SchemaVertexTypeUtilities matchVertexTypes
WARNING: Multiple types matched identifier 192.168.1.1: [Network Identifier.IP Address.IPv4 Address, Online Location.Host Name]
```

Note that if I run the 'Extract Types from Text" plugin and put in 192.168.1.1 I get "IPv4 Address" for the type which is inconsistent to Hostname here. The issue there looks like different methods are used. Split nodes uses `SchemaVertexTypeUtilities.matchVertexTypes()` and the applies the `VertexDominantCalculator` whereas "Extract Types from Text" plugin uses `SchemaVertexTypeUtilities.extractVertexTypes()`.

I would suggest this is a bug and should be resolved asap but outside the scope of how Split Nodes plugin works

Reply:
@cygnus-x-1 I created #596 to continue this discussion. I'm going to merge this PR to fix the original bug and unit test.